src/components/algolia-search/algolia-search.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  margin-bottom: 34px;
+}
+
+.form {
+  border: none;
+  box-shadow: var(--token-surface-inset-box-shadow);
+  border-radius: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+
+  &:focus-within {
+    outline: none;
+
+    /* --token-color-focus-action-internal */
+
+    /* border: 1px solid var(--token-color-palette-blue-300); */
```

Comment:
I've got to nail this down still. The designs show a border around the search element (which would go on `.form`), but it behaves strangely in conjunction with the box-shadows and focus interaction.